src/components/elements/hero.js

```diff
@@ -12,14 +12,15 @@ const Wrapper = styled.section`
 const StyledHeader = styled(Header)`
     padding: 0.8rem 0;
     width: ${props => props.paragraph_width || ''};
+    max-width: ${props => props.paragraph_max_width || ''};
 `
 
 const NoImageHero = styled(Container)`
     flex-direction: column;
     align-content: space-around;
     padding: 8rem 0 10rem 0;
 `
-const Hero = ({ header, paragraph, paragraph_width }) => (
+const Hero = ({ header, paragraph, paragraph_width, paragraph_max_width }) => (
```

Comment:
is it possible to use either max_width or width instead of both

Reply:
yes, sometimes we need to set a limit on it.